### PR TITLE
[Docs] Update Japanese translation of faq related documents

### PR DESCRIPTION
**Pull request by shelaf**
## Description

Update Japanese translation of faq related documents

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@umi-umi @s-show @hsgw よかったらレビューお願いします。
以前の構成をすこし組み替えただけのようにもみえます。訳の新しい部分はないかもしれません。

File: docs/ja/faq_general.md [hunk 1]
```diff
@@ -1,14 +1,52 @@
...
+
+## サポートを追加したいキーボードがあります
+
+素晴らしい！プルリクエストを開いてください。私たちはコードをレビューし、マージします！
+
+### `QMK` でブランドしたい場合はどうればいいですか？
+
+素晴らしい！私たちはあなたを支援したいと思います！
+
+実際、私たちはあなたのページとキーボードに QMK ブランドを追加するための [完全なページ](https://qmk.fm/powered/) があります。これは QMK を公式にサポートするために必要なほぼ全て(知識と画像)をカバーしています。
```
Comment (s-show) [inline docs/ja/faq_general.md]: 「私たちは〜があります」だと主語と述語が合わない感じがしますので、「私たちには〜があります」の方が良いかなと思います。
```suggestion
実際、私たちにはあなたのページとキーボードに QMK ブランドを追加するための [完全なページ](https://qmk.fm/powered/) があります。これは QMK を公式にサポートするために必要なほぼ全て(知識と画像)をカバーしています。
```

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks !

**Pull request merged**